proj/scalastyle.conf

```diff
@@ -1,9 +0,0 @@
-// https://github.com/scalastyle/scalastyle.git#master
```

Comment:
Hi Seth, Scalastyle hasn't been maintained for a long time (https://github.com/scalastyle/scalastyle/issues/327) and the maintainers are no longer active, so I decided to [fork](https://github.com/beautiful-scala/scalastyle) the project and [release](https://github.com/beautiful-scala/scalastyle/releases/tag/v1.1.0) a new version with support for 2.13. I also updated and published the [website](https://scalastyle.beautiful-scala.com).

Would it be possible to add this project to the community build instead?

Reply:
Sure, let's give it a try: https://github.com/scala/community-build/pull/1055